plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/rest/TestIcebergLakekeeperCatalogConnectorSmokeTest.java

```diff
@@ -0,0 +1,393 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.iceberg.catalog.rest;
+
+import io.trino.filesystem.FileIterator;
+import io.trino.filesystem.Location;
+import io.trino.plugin.iceberg.BaseIcebergConnectorSmokeTest;
+import io.trino.plugin.iceberg.IcebergConfig;
+import io.trino.plugin.iceberg.IcebergConnector;
+import io.trino.plugin.iceberg.IcebergQueryRunner;
+import io.trino.plugin.iceberg.catalog.TrinoCatalog;
+import io.trino.plugin.iceberg.catalog.TrinoCatalogFactory;
+import io.trino.plugin.iceberg.fileio.ForwardingFileIo;
+import io.trino.spi.connector.SchemaTableName;
+import io.trino.testing.QueryRunner;
+import io.trino.testing.TestingConnectorBehavior;
+import org.apache.iceberg.BaseTable;
+import org.apache.iceberg.TableMetadata;
+import org.apache.iceberg.TableMetadataParser;
+import org.apache.iceberg.io.FileIO;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.parallel.Isolated;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+
+import static io.trino.plugin.iceberg.IcebergTestUtils.*;
+import static io.trino.testing.TestingNames.randomNameSuffix;
+import static java.lang.String.format;
+import static org.apache.iceberg.FileFormat.PARQUET;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+import static org.junit.jupiter.api.TestInstance.Lifecycle.PER_CLASS;
+
+@Isolated // TODO remove
+@TestInstance(PER_CLASS)
+final class TestIcebergLakekeeperCatalogConnectorSmokeTest
+        extends BaseIcebergConnectorSmokeTest {
+    private TestingLakekeeperCatalog lakekeeperCatalog;
+
+    public TestIcebergLakekeeperCatalogConnectorSmokeTest() {
+        super(new IcebergConfig().getFileFormat().toIceberg());
+    }
+
+    @Override
+    protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior) {
+        return switch (connectorBehavior) {
+            case SUPPORTS_CREATE_MATERIALIZED_VIEW,
+                 SUPPORTS_RENAME_SCHEMA -> false;
+            default -> super.hasBehavior(connectorBehavior);
+        };
+    }
+
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception {
+        lakekeeperCatalog = closeAfterClass(new TestingLakekeeperCatalog("125ms"));
```

Comment:
Why do we use a different poll interval between `TestIcebergLakekeeperCatalogConnectorSmokeTest` and `IcebergLakekeeperQueryRunnerMain`?

Reply:
There are a few tests that check that files are gone after dropping. Since Lakekeeper cleans up files async and returns a response before the clean-up is done, we need to sleep in the tests. The polling interval determines how long we have to sleep. A smaller poll interval means tests run faster but it also makes the Lakekeeper instance chattier. In `IcebergLakekeeperQueryRunnerMain`,  we dump the logs if an exception occurs, if we have a small poll interval there, the logs become quite unreadable, at the same time, we don't need the almost-instant cleanup in `IcebergLakekeeperQueryRunnerMain` since there are no tests checking if files are actually gone.